From 321bb01bd142632a41c40935206ac1e1c8a2e3c2 Mon Sep 17 00:00:00 2001
From: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Date: Tue, 11 Jun 2024 16:55:18 +0200
Subject: [PATCH] ucm2: codecs: wcd938x: correct 'cset' command

Intention was to use 'cset':

  ALSA lib parser.c:1224:(parse_sequence) error: sequence command 'set' is ignored

Closes: https://github.com/alsa-project/alsa-ucm-conf/pull/423
Signed-off-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/codecs/wcd938x/HeadphoneMicDisableSeq.conf | 2 +-
 ucm2/codecs/wcd938x/HeadphoneMicEnableSeq.conf  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ucm2/codecs/wcd938x/HeadphoneMicDisableSeq.conf b/ucm2/codecs/wcd938x/HeadphoneMicDisableSeq.conf
index 9021b92..47f242d 100644
--- a/ucm2/codecs/wcd938x/HeadphoneMicDisableSeq.conf
+++ b/ucm2/codecs/wcd938x/HeadphoneMicDisableSeq.conf
@@ -1,5 +1,5 @@
 DisableSequence [
 	cset "name='ADC2_MIXER Switch' 0"
 	cset "name='ADC2 Switch' 0"
-	set "name='TX1 MODE' ADC_INVALID"
+	cset "name='TX1 MODE' ADC_INVALID"
 ]
diff --git a/ucm2/codecs/wcd938x/HeadphoneMicEnableSeq.conf b/ucm2/codecs/wcd938x/HeadphoneMicEnableSeq.conf
index f3a7807..de5a759 100644
--- a/ucm2/codecs/wcd938x/HeadphoneMicEnableSeq.conf
+++ b/ucm2/codecs/wcd938x/HeadphoneMicEnableSeq.conf
@@ -3,5 +3,5 @@ EnableSequence [
 	cset "name='HDR12 MUX' NO_HDR12"
 	cset "name='ADC2 MUX' INP2"
 	cset "name='ADC2 Switch' 1"
-	set "name='TX1 MODE' ADC_NORMAL"
+	cset "name='TX1 MODE' ADC_NORMAL"
 ]
-- 
2.34.1

